src/util.cpp

```diff
@@ -3602,6 +3602,35 @@ static std::unordered_map<std::string,int> g_usedNames;
 static std::mutex g_usedNamesMutex;
 static int g_usedNamesCount=1;
 
+/*! This function converts the value of CREATE_SUBDIRS
+ *  into the number of relevant bits for the number of
+ *  created directories in the 2nd level.
+ */
+static int convertCreateSubDirs2Level(CREATE_SUBDIRS_t createSubDirs)
+{
+  switch (createSubDirs)
+  {
+    case CREATE_SUBDIRS_t::D16:
+      return 0;
```

Comment:
Doesn't this mean that in this case there is only one sub-directory at level 2, I think this is a bit overdone (even `D32`  and `D64` might be a bit overdone).